src/main/java/de/tum/in/www1/artemis/web/rest/OrganizationResource.java

```diff
@@ -146,7 +147,7 @@ public OrganizationResource(OrganizationService organizationService, Organizatio
      * @param organization the updated organization entity
      * @return the ResponseEntity containing the updated organization with status 200 (OK), or 404 (Not Found) otherwise
      */
-    @PutMapping("/organizations/update")
+    @PutMapping("/organizations/{organizationId}")
```

Comment:
I would argue you do, I first want to check the consistency of the request and whether the client knows which object is being changed. Also, the RFC for REST demands for PUT-requests to be executed on the same path as a GET-request would be on the updated resource, and since GET would take an Id here, you also need PUT to take the ID

Reply:
Alright, I see. Then shouldn't there be a check in the method body whether organizationId equals organization.getId() or this is not needed?

Reply:
Oh, and one minor thing, the JavaDoc still has the old URL